charts/kube-state-metrics/templates/role.yaml

```diff
@@ -199,6 +199,20 @@ rules:
     - subjectaccessreviews
   verbs: ["create"]
 {{- end }}
+{{- if $.Values.customResourceState.enabled }}
+- apiGroups: ["apiextensions.k8s.io"]
+  resources:
+    - customresourcedefinitions
+  verbs: ["list", "watch"]
+{{- if $.Values.customResourceState.generateRBAC }}
+{{- range $.Values.customResourceState.config.spec.resources }}
```

Comment:
This unfortunately does not work as expected as the clusterrole expects a resource and not a kind.

Reply:
I think for right now, I would drop it and point to .Values.rbac.extraRules